src/ui/canvas/gdi/GdiPlusBitmap.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2021 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#include "GdiPlusBitmap.hpp"
+
+#if defined(_MSC_VER)
+# include <algorithm>
+# define min std::min
+# define max std::max
+#endif  // _MSC_VER
```

Comment:
Why?

Reply:
Inside the gdiplus headers the 'min' and 'max' are using very often, but with the define NOMINMAX they are not available. So you have define it explicitly. The recommandations found in stackoverflow (https://stackoverflow.com/questions/4913922/possible-problems-with-nominmax-on-visual-c) shows exactly this problem, but they are using the 'using' directive, not the #define - and so I changed it in this way.